integration/integration_test.go

```diff
@@ -186,6 +190,106 @@ func TestDownloadSyncthing(t *testing.T) {
 	}
 }
 
+func TestStacks(t *testing.T) {
+	if mode == "client" {
+		return
+	}
+
+	ctx := scopeagent.GetContextFromTest(t)
+	oktetoPath, err := getOktetoPath(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tName := fmt.Sprintf("TestStacks-%s", runtime.GOOS)
+	name := strings.ToLower(fmt.Sprintf("%s-%d", tName, time.Now().Unix()))
+	namespace := fmt.Sprintf("%s-%s", name, user)
+
+	test := scopeagent.GetTest(t)
+	test.Run(tName, func(t *testing.T) {
+		if err := createNamespace(ctx, oktetoPath, namespace); err != nil {
+			t.Fatal(err)
+		}
+
+		defer deleteNamespace(ctx, oktetoPath, namespace)
```

Comment:
Could we not delete it if it fails? It's helpful for debugging. We can have a once week job that deleted any leftover namespaces.